ee/clickhouse/sql/funnels/funnel.py

```diff
@@ -55,6 +55,16 @@
 ;
 """
 
+FUNNEL_PERSONS_BY_STEP_SQL = """
+SELECT person_id
+FROM
+({steps_per_person_query})
+WHERE {persons_steps}
+LIMIT 100
```

Comment:
I think we need an order by here to ensure `offset` works properly / gives deterministic results?

Reply:
Otherwise, this is good, general enough to be used in every kind of funnel we have :D

Reply:
we should do this ordering for all the people pagination queries